java/ql/src/semmle/code/java/security/XPath.qll

```diff
@@ -0,0 +1,64 @@
+/** Provides classes to reason about XPath vulnerabilities. */
+
+import java
+import semmle.code.java.dataflow.FlowSources
+import semmle.code.java.dataflow.TaintTracking
+
+/** The interface `javax.xml.xpath.XPath` */
```

Comment:
```suggestion
/** The interface `javax.xml.xpath.XPath`. */
```

Reply:
In aa10a5d, I refactored the library to use the CSV sink format, so this comment was removed.